src/pkg/services/m365/api/lists_test.go

```diff
@@ -328,9 +328,10 @@ func (suite *ListsUnitSuite) TestColumnDefinitionable_LegacyColumns() {
 	roCd.SetReadOnly(ptr.To(true))
 
 	tests := []struct {
-		name    string
-		getList func() *models.List
-		length  int
+		name             string
+		getList          func() *models.List
+		length           int
+		expectedColNames map[string]struct{}
```

Comment:
and here throughout the tests
```suggestion
		expectedColNames map[string]any
```